src/components/PassphraseForm.jsx

```diff
@@ -0,0 +1,56 @@
+import styles from '../styles/passphraseForm'
+
+import React from 'react'
+import stateFulPassphraseForm from '../lib/stateFulPassphraseForm'
+
+const PassphraseForm = ({ t, children, submitting, currentPassphrase, newPassphrase, submitPassphrase, isFormValid }) => (
+  <div class={styles['coz-form']}>
+    <h3>{t('AccountView.password.title')}</h3>
+    <label>{t('AccountView.password.current_label')}
+      <a
+        onClick={currentPassphrase.toggleVisibility}
+        class={styles['visibility']}
+      >
+        {currentPassphrase.visible ? 'Hide' : 'Show'}
+      </a>
+    </label>
+    <input
+      type={currentPassphrase.visible ? 'text' : 'password'}
+      placeholder={t('AccountView.password.current_placeholder')}
+      value={currentPassphrase.value}
```

Comment:
I found this way more explicit here and meaningful, even it could be lighter in your way. What do others think? @gregorylegarec ? @m4dz ? @GoOz ?